packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -1163,6 +1177,12 @@ abstract class DatabaseClusterNew extends DatabaseClusterBase {
       throw new ValidationError('serverlessV2MinCapacity & serverlessV2MaxCapacity must be in 0.5 step increments, received '+
       `min: ${this.serverlessV2MaxCapacity}, max: ${this.serverlessV2MaxCapacity}`, this);
     }
+
+    if (this.autoPause && !this.autoPause.isUnresolved()) {
+      if (this.autoPause.toSeconds() < 300 || this.autoPause.toSeconds() > 86400) {
```

Comment:
IMO: Nesting of `if` statements should be avoided if possible. (Since there are no other validations for this property.)

```suggestion
    if (this.autoPause && !this.autoPause.isUnresolved() && (this.autoPause.toSeconds() < 300 || this.autoPause.toSeconds() > 86400)) {
```